tools/eksDistroBuildToolingOpsTools/Makefile

```diff
@@ -1,14 +1,35 @@
 GIT_VERSION?=v0.0.0-dev
+IMAGE_BUILD_ID=$(shell cat ./EKS_DISTRO_PROW_PLUGIN_RELEASE)
+
+AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
+
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+PROJECT_DIRECTORY:=$(BASE_DIRECTORY)/tools/eksDistroBuildToolingOpsTools
+
+# This image build is specifically focused on a debian base image for use with EKS build systems
+IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+PUSH_IMAGES?=true
+
+GITHUB_EMAIL?="prow@amazonaws.com"
+GITHUB_USER?="Prow Bot"
 
 GO_ARCH:=$(shell go env GOARCH)
 GO_OS:=$(shell go env GOOS)
 
 EKS_GO_TOOLS_BINARY_NAME="eksGoTools"
+EKS_PROW_PLUGIN_BINARY_NAME="eksDistroOpsProwPlugin"
 
 .PHONY: build-eks-go-tools
-build-eks-go-tools:  unit-test lint
+build-eks-go-tools:
 	CGO_ENABLED=0 GOOS=$(GO_OS) GOARCH=$(GO_ARCH) go build -ldflags "-X github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/version.gitVersion=$(GIT_VERSION) -s -w -extldflags -static" -o bin/$(GO_OS)/$(GO_ARCH)/$(EKS_GO_TOOLS_BINARY_NAME) github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool
 
+.PHONY: build-prowPlugin
+build-prowPlugin: 
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(GO_ARCH) go build -ldflags "-X github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/version.gitVersion=$(GIT_VERSION) -s -w -extldflags -static" -o bin/$(GO_OS)/$(GO_ARCH)/$(EKS_PROW_PLUGIN_BINARY_NAME) github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/cmd/$(EKS_PROW_PLUGIN_BINARY_NAME)
+
+.PHONY: images-eksDistroOpsProwPlugin
```

Comment:
let's change up these target names a bit...here's my recommendation

top level commands:
`build-eksDistroProwPlugin` and `release-eksDistroProwPlugin`

these both then invoke `build-binary-eksDistroProwPlugin` which actually runs the go build command, runs unit tests, runs the go lint. 

then, the `build-eksDistroProwPlugin` calls `local-images-eksDistroProwPlugin`; and the `release-eksDistroProwPlugin` target calls `images-eksDistroProwPlugin`.

If we do end up bundling the prow plugin functionality into the ops tool binary like we discussed, rather than having it be a seperate binary, we can use the same strategy, we will probably just want to rename the target `build-binary-eksDistroProwPlugin` to `build-binary-eksDistroOpsTool` so that it's consistent; we can still use the `eksDistroProwPlugin` names for the rest of the targets, so that it's clear what we're doing, since the image publishing and building etc will be specific to the plugin. Thoughts?